src/fty_metric_composite_configurator.cc

```diff
@@ -35,13 +35,11 @@
 static const char *AGENT_NAME = "fty-metric-composite-configurator";
```

Comment:
could have used the composite prefix, or defined some complement if needed
https://github.com/42ity/fty-common/blob/master/include/fty_common_agents.h#L41